headers/Application.hpp

```diff
@@ -0,0 +1,46 @@
+#include <vector>
+#include <random>
+
+#include "SDL.h"
+#include "SDL_image.h"
+#include "SDL_ttf.h"
+
+#include "Renderer.hpp"
+
+#include "Scene.hpp"
+#include "Scenes/GamePlayScene.hpp"
+
+#define DEBUG_OUTPUT true
+#define GHOSTS 896*20
+
+class Application {
+	public:
+		Application();
+		void init();
+		void eventHandling();
+		void render();
+		void update();
+		void run();
```

Comment:
We should include methods to push/pop methods from the Scene stack, allowing gameplay to be passed to other scenes. Make sure we update `currentScene` when we do this.